x/auth/spec/05_vesting.md

```diff
@@ -426,6 +498,89 @@ Same initial starting conditions as the simple example.
 
     Notice how we have an excess amount of `DV`.
 
+### Periodic Vesting
+
+A vesting account is created where 100 tokens will be released over 1 year, with 1/4 of tokens vesting each quarter. The vesting schedule would be as follows:
+
+```json
+{
+  "vesting_periods": [
+    {
+      "period_length": 7884000,
+      "vesting_amount": [
+        {
+        "denom": "stake",
+        "amount": "25"
+      }
+    ]
+    },
+    {
+      "period_length": 7884000,
+      "vesting_amount": [
+        {
+        "denom": "stake",
+        "amount": "25"
+      }
+    ]
+    },
+    {
+      "period_length": 7884000,
+      "vesting_amount": [
+        {
+        "denom": "stake",
+        "amount": "25"
+      }
+    ]
+    },
+    {
+      "period_length": 7884000,
+      "vesting_amount": [
+        {
+        "denom": "stake",
+        "amount": "25"
+      }
+    ]
+    }
+  ]
+}
+```
+
+```
+OV = 100
+DF = 0
+DV = 0
+BC = 100
+V = 100
+V' = 0
+```
+
+1. Immediately receives 1 coin
+
+    ```
+    BC = 101
+    ```
+
+2. Vesting period 1 passes, 25 coins vest
+
+    ```
+    V = 75
+    V' = 25
+    ```
+
+3. During vesting period 2, 5 coins are transfered and 5 coins are delegated
```

Comment:
It's been a while since I wrote this, but having trouble following along here, so correct me if I'm wrong.

At this point, assuming we're in period 2 and it has not passed, then:

```
X ← min(max(75 - 0, 0), 5)
  ← min(max(75, 0), 5)
  ← min(75, 5)
  ← 5

DV ← DV + 5
   ← 5
```